lib/zooniverse-react-components.css

```diff
@@ -122,3 +122,6 @@
   margin-right: 0.25em;
   white-space: nowrap;
 }
+.paginator .paginator-page-selector select {
```

Comment:
No, I've setup preversion and postversion scripts (read more here: https://docs.npmjs.com/cli/version). The preversion will build the transpiled js and css. I haven't added the lib folder to gitignore yet since I realize during the early development of this repository it might be helpful to have it available for testing with installation via github branch, but once we release a v1 of this on npm, the lib folder should be ignored to make sure we're getting clean builds.